### ISSUE TITLE
back up / export?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [arkayenro/arkinventory#679]
```
i.  love.  this.  addon.  Ark is the best addon ever, i love it. i've spent a 
ton of time setting up my rules and custom categories, hours programming this 
addon and it's glorious.  

now, how do i back it up?  say i play wow on another computer, i want to be 
able to transfer all my rules and settings right over, are there certain files 
i need to copy?  file locations?

i've look over this site for awhile now and the only thing i found related to 
this was back in 2012, someone was wanting an export feature, so i'm making a 
post.  thanks for the great addon!  
```

Original issue reported on code.google.com by `thekings...@gmail.com` on 6 Dec 2014 at 11:23

**Comment by arkayenro** [comment]
closing old issues - if its still a problem re-open the ticket

essentially whats been said, backup your savedvariables file, or the whole WTF folder preferably

**Comment by spiralofhope** [comment]
See also issue #74 (Ability to export profile), implemented in 3.06.04
